### DISCUSSION TITLE
关于伪装站的问题，我看了下配置没发现有回落的选项

### DISCUSSION THREAD

**Discussion by picklefan** [apernet/hysteria#418] [Q&A]
我感觉是GFW 目前没有针对quic 的主动嗅探，被封的应该都是因为行为特征（大流量、长连接）

**Comment by picklefan** [comment]
看到了这个issue #73，看来是目前比较难实现的功能。不过话说回来，网上找不到几个说用hysteria 被墙的人，到底是墙懒得封，还是quic 协议有什么奇淫技巧。我感觉发请求给域名一直404还天天有流量应该很快被墙呀，这不科学

**Reply by white586** [reply]
我的搭建好了 用chrome（支持http3）访问https的域名端口直接无法显示正常吗？不是404。很担心被墙发现，伪装不是很好

**Comment by picklefan** [comment]
速度确实快，比我以前的xtls-splice快了30%左右，可惜只能坚持一天左右
